src/test/run-promtool-unittests.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+promtool test rules /ceph/monitoring/prometheus/alerts/test_alerts.yml
```

Comment:
At this point it should probably then try to run the test using `promtool` of the container like so:

```bash
docker run --rm -it --entrypoint promtool prom/prometheus:v2.9.2 test rules PATH
```

where PATH is the path to the `test_alerts.yml` file inside the container. For this to succeed, `promtool` would also need to find `ceph_default_alerts.yml` next to `test_alerts.yml` inside the container. If it isn't already in the container (which, according to the state of this pull request,  I do not _assume_ is the case), then you'd need to map the file to the container when starting the container with `-v SOURCE_FILE:TARGET_FILE:ro`. Although promtool should not change the content of any file mapped to the container, it might still be appropriate to set the file to read-only using the `:ro` suffix.

So, you'll probably end up with something like the following. Note that I have not tested this code.

```bash
docker run --rm -it \
  --entrypoint promtool \
  -v /ceph/monitoring/prometheus/alerts:/alerts:ro \
  test rules /alerts/test_alerts.yml
```

Reply:
Thanks @p-se ..But the docker command is also not getting executed inside the script..It shows `docker:command not found`.

Reply:
By pulling the pormtool container in the `install-deps.sh`, I could see the traces of the container in the logs..but still got `promtool: command not found` error in the make-check.